pkg/scheduler/core/util.go

```diff
@@ -77,11 +77,12 @@ func calAvailableReplicas(clusters []*clusterv1alpha1.Cluster, spec *workv1alpha
 
 // findOutScheduledCluster will return a name set of clusters
```

Comment:
The comments should be updated as well. Sorry for missing it from my last comment.